imports/ui/pages/professor/manage_session.jsx

```diff
@@ -225,6 +225,8 @@ class _ManageSession extends Component {
       plainText: '', // plain text version of question
```

Comment:
I think we should take out this blank question and have it live in question.js, just like we do for default_session_options, export it from there and import it here. That way, we don't have multiple locations to update if we change the structure of a question.

Reply:
I'm wondering if I should just take the default_state from `QuestionEditItem.jsx` and either import it from there or move it to `questions.js`?

Reply:
I think we should move that default question to question.js. In fact, we may want to refactor QuestionEditItem a little so that its state is not a question, but rather that it has a question in its state (and that can be the default question, perhaps with default session options in it).

Reply:
What I have done for now is moved the default question to `questions.js`, and imported it into `QuestionEditItem.jsx` so that its state just extends the default question if making a new question.